Makefile

```diff
@@ -485,9 +485,16 @@ check-scripts: ## Check that scipts do not have common mistakes
 check-deny: ## Check advisories licenses and sources for crate dependencies
 	${MAYBE_ENVIRONMENT_EXEC} ./scripts/check-deny.sh
 
+.PHONY: check-events
 check-events: ## Check that events satisfy patterns set in https://github.com/vectordotdev/vector/blob/master/rfcs/2020-03-17-2064-event-driven-observability.md
 	${MAYBE_ENVIRONMENT_EXEC} ./scripts/check-events
 
+ifeq (${CI}, true)
+.PHONY: check-component-docs
+check-component-docs: generate-component-docs ## Checks that the machine-generated component Cue docs are up-to-date.
+	${MAYBE_ENVIRONMENT_EXEC} ./scripts/check-component-docs.sh
+endif
```

Comment:
Why does this need to be guarded here? Wouldn't it be harmless to run outside of CI?

Reply:
Hmmm, I suppose after thinking about it, running it locally _would_ be fine. That way, too, I could include it in the default "check" target.

I'll change this. 👍🏻